sunpy/coordinates/tests/test_frames.py

```diff
@@ -427,3 +427,20 @@ def test_skycoord_hpc(args, kwargs):
         hgs = sc.transform_to("heliographic_stonyhurst")
 
     assert isinstance(hgs.frame, HeliographicStonyhurst)
+
+
+def test_angular_radius():
+    coord = Helioprojective(0*u.deg, 0*u.deg, 5*u.km, obstime="2010/01/01T00:00:00", observer="earth")
+    assert_quantity_allclose(coord.angular_radius, sun.angular_radius(coord.obstime))
```

Comment:
This is failing:
```python
[gw0] linux -- Python 3.9.5 /home/vsts/work/1/s/.tox/py39/bin/python

    def test_angular_radius():
        coord = Helioprojective(0*u.deg, 0*u.deg, 5*u.km, obstime="2010/01/01T00:00:00", observer="earth")
>       assert_quantity_allclose(coord.angular_radius, sun.angular_radius(coord.obstime))
E       AttributeError: module 'sunpy.sun' has no attribute 'angular_radius'
```

Reply:
`sun` here resolves to `sunpy.sun`, which is not the same as `sunpy.coordinates.sun`.  You should just import `angular_radius()` from `sunpy.coordinates.sun` and call it directly.